src/uct/api/v2/uct_v2.h

```diff
@@ -784,7 +784,10 @@ typedef enum uct_md_attr_field {
     UCT_MD_ATTR_FIELD_EXPORTED_MKEY_PACKED_SIZE = UCS_BIT(14),
 
     /** Unique global identifier of the memory domain. */
-    UCT_MD_ATTR_FIELD_GLOBAL_ID                 = UCS_BIT(15)
+    UCT_MD_ATTR_FIELD_GLOBAL_ID                 = UCS_BIT(15),
+
+    /** Indicate minimal registration granularity. */
```

Comment:
`indicate minimal registration granularity ` - I don't think granularity and alignment have the same meaning.